docs/design/coreclr/botr/readytorun-format.md

```diff
@@ -75,11 +92,28 @@ MinorVersion increments are meant to compatible file format changes.
 successfully execute native code from images of version 2.9. The runtime should refuse to execute
 native code from image of version 3.0.
 
-### READYTORUN_HEADER::Flags
+## READYTORUN_CORE_HEADER
+
+```C++
+struct READYTORUN_CORE_HEADER
+{
+    DWORD                   Flags;          // READYTORUN_FLAG_XXX
+
+    DWORD                   NumberOfSections;
+
+    // Array of sections follows. The array entries are sorted by Type
+    // READYTORUN_SECTION   Sections[];
+};
+```
+
+### READYTORUN_CORE_HEADER::Flags
 
 | Flag                                    |      Value | Description
 |:----------------------------------------|-----------:|:-----------
 | READYTORUN_FLAG_PLATFORM_NEUTRAL_SOURCE | 0x00000001 | Set if the original IL image was platform neutral. The platform neutrality is part of assembly name. This flag can be used to reconstruct the full original assembly name.
+| READYTORUN_FLAG_COMPOSITE               | 0x00000002 | The image represents a composite R2R file resulting from a combined compilation of a larger number of input MSIL assemblies.
+| READYTORUN_FLAG_EMBEDDED_MSIL           | 0x00000004 | Input MSIL is embedded in the R2R image.
+| READYTORUN_FLAG_COMPONENT               | 0x00000008 | This is 
```

Comment:
Unfinished sentence?

Reply:
Thanks for spotting, fixed in 3rd commit.